hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OutboundResponseHandler.java

```diff
@@ -98,4 +114,39 @@ public boolean send(Response response, Address target) {
         Connection connection = connectionManager.getOrConnect(target);
         return connectionManager.transmit(packet, connection);
     }
+
+    public void sendBackupAck(Address target, long callId, boolean urgent) {
+        checkNotNull(target, "Target is required!");
+
+        if (thisAddress.equals(target)) {
+            throw new IllegalArgumentException("Target is this node! -> " + target);
+        }
+
+        byte[] bytes = new byte[BACKUP_RESPONSE_SIZE_IN_BYTES];
+
+        // the first 4 bytes contain the partition hash which are not set.
+
+        //serializer type id
+        writeInt(bytes, OFFSET_SERIALIZER_TYPE_ID, CONSTANT_TYPE_DATA_SERIALIZABLE, useBigEndian);
+        // identified
+        bytes[OFFSET_IDENTIFIED] = 1;
+        // factory id
+        writeInt(bytes, OFFSET_TYPE_FACTORY_ID, SpiDataSerializerHook.F_ID, useBigEndian);
+        // type id
+        writeInt(bytes, OFFSET_TYPE_ID, BACKUP_ACK_RESPONSE, useBigEndian);
+        //call id
+        writeLong(bytes, OFFSET_CALL_ID, callId, useBigEndian);
+
+        Packet packet = new Packet(bytes, -1)
```

Comment:
Maybe you can pull out this part as `createPacket(bytes, urgent)` so there is no code duplication in this class.